### DISCUSSION TITLE
Tapo H100 Smart Hub issue

### DISCUSSION THREAD

**Discussion by Piksanhome** [petretiandrea/home-assistant-tapo-p100#334] [Q&A]
I could connect H100 successfully. May I know if it is possible to connect the device under the hub, such as Tapo S220? Thank you.

**Comment by KennethS2022** [comment]
> I could connect H100 successfully. May I know if it is possible to connect the device under the hub, such as Tapo S220? Thank you.

I also am hoping this is possible. The integration works well connecting to the H100 but being able to control the S220 is what ideally I need.
